examples/solids/index.rst

```diff
@@ -700,6 +700,14 @@ Collecting terms, the weak form of the Newton linearization for Neo-Hookean mate
 which equivalent to Algorithm 2 of :cite:`davydov2020matrix` and requires only derivatives with respect to the current configuration. Note that :math:numref:`cur_simp_Jac` and :math:numref:`jacobian-weak-form-current2` have recovered the same representation 
 using different algebraic manipulations. 
 
+.. tip::
+   An alternative form of Kirchhoff stress tensor, :math:`\bm{\tau}`, can be achieved by rewriting :math:`\bm{b} - \bm{I}_3` in terms of :math:`\nabla_X \bm{u}` in :math:numref:`tau-neo-hookean` as
+
+   .. math::
+      \bm \tau = \mu \Big( \nabla_X \bm{u} + (\nabla_X \bm{u})^T + \nabla_X \bm{u} (\nabla_X \bm{u})^T \Big) + \lambda \log J \bm I_{3}.
+
+   which is more numerically stable for small strain, and thus preferred for computation.
```

Comment:
This looks fine, thanks. Rich's suggestion was to call `(b - I)/2` the Green-Euler strain (because it's defined in the Eulerian frame, versus Green-Lagrange `E = (C - I)/2` being defined in the Lagrangian frame).

Reply:
Please, check the last commit. If everything is fine, we can merge this branch.